client/components/select-vertical/index.tsx

```diff
@@ -76,7 +76,8 @@ const SelectVertical: React.FC< Props > = ( {
 						: []
 				}
 				isLoading={ isDebouncing || isLoadingDefaultVertical || isLoadingSuggestions }
-				isShowSkipOption={ hasUserInput }
+				// @TODO: Finally remove if not used becasue now we are forcing it as true for a quick try
+				isShowSkipOption
```

Comment:
@miksansegundo let's remove this option entirely, since the "Something else" option will show in every scenario.

Reply:
Done ✅